container/runDockerContainer.sh

```diff
@@ -23,17 +24,41 @@ language_image_name="${DEFAULT_IMAGE_PREFIX}${LANGUAGE_ID}"
 language_image_tag="${language_image_name}:${language_image_version}"
 
 echo "Running ${language_image_tag} from the local docker registry"
-docker run                                                                      \
-      --env AWS_ACCESS_KEY_ID=unused                                            \
-      --env AWS_SECRET_KEY=unused                                               \
-      --env S3_ENDPOINT=unused                                                  \
-      --env S3_REGION=unused                                                    \
-      --env SQS_ENDPOINT=unused                                                 \
-      --env SQS_REGION=unused                                                   \
-      --env SQS_QUEUE_URL=unused                                                \
-      --env PARTICIPANT_ID=${PARTICIPANT_ID}                                    \
-      --env ROUND_ID=${ROUND_ID}                                                \
-      --env REPO=${REPO}                                                        \
-      --env TAG=${TAG}                                                          \
-      --env CHALLENGE_ID=${CHALLENGE_ID}                                        \
-      ${language_image_tag}
+if [[ "${DEBUG}" == "true" ]]; then
```

Comment:
There is a lot of duplication between the IF branches. The if could set a variable with only the differences.